.github/workflows/compilation.yml

```diff
@@ -33,3 +33,26 @@ jobs:
         export PS2DEV=$PWD/ps2dev
         export PATH=$PATH:$PS2DEV/dvp/bin
         ./toolchain.sh
+
+  build-win:
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: msys2 {0}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install MSYS2 texinfo bison flex
+      uses: msys2/setup-msys2@v2
+      with:
+        msystem: MINGW32
+        install: git msys2-devel base-devel binutils mingw-w64-i686-gcc binutils git unzip make texinfo tar patch mpc-devel p7zip man bison flex
```

Comment:
most probably, to compile just binutils for `dvp`, a lot of libraries from here aren't required, can you just try to reduce to the minimum?

Reply:
just realized that I pasted `binutils` 2 times :D . Yes now it takes a lot of time to pass through the package installation stage.

Reply:
If you take a look to the `Dockerfile` you can really see what are the minimum libraries required by `Alpine`, somehow they should be similar to the required by windows (or even less because virtual machines used to come with more libraries installed by default)

```
RUN apk add build-base git bash patch wget texinfo gettext
```